.publicApi/Microsoft.ApplicationInsights.WorkerService.dll/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,4 @@
+Microsoft.ApplicationInsights.WorkerService.ApplicationInsightsServiceOptions.EnableDiagnosticsTelemetryModule.get -> bool
+Microsoft.ApplicationInsights.WorkerService.ApplicationInsightsServiceOptions.EnableDiagnosticsTelemetryModule.set -> void
+Microsoft.ApplicationInsights.WorkerService.ApplicationInsightsServiceOptions.EnableTelemetryConfigurationActiveBackwardsCompatibility.get -> bool
```

Comment:
Shouldn't need this as WorkerService never populated Active, and no need to introduce that behavior heere.